lib/compress.js

```diff
@@ -1731,6 +1731,7 @@ merge(Compressor.prototype, {
                 var val = {};
                 for (var i = 0, len = this.properties.length; i < len; i++) {
                     var prop = this.properties[i];
+                    if (prop instanceof AST_Expansion) return this;
```

Comment:
No problem - I was waiting for you to stop and let me hit that "merge" button :rofl: